python/ml4ir/base/model/relevance_model.py

```diff
@@ -778,3 +779,11 @@ def _build_callback_hooks(
         # Add more here
 
         return callbacks_list
+
+    def calibration(self, relevance_dataset, logger, logs_dir_local, temperature_init):
+        temperature_scale(model=self.model,
```

Comment:
@marjanhs You can maybe add an info log here saying that you are calibrating the model.

Reply:
added

Reply:
thanks!